QUEST_LV_0100.tsv

```diff
@@ -5826,7 +5826,7 @@ QUEST_LV_0100_20150317_005825	Check the stone pillars of Alkune Uphill
 QUEST_LV_0100_20150317_005826	Now that you have taken care of the monsters, check the stones pillars around Alkune Uphill. You will be able to find traces of relics in the pillars.
 QUEST_LV_0100_20150317_005827	Retrieved all the relics of Raudi. Go back and give the relics to Raune.
 QUEST_LV_0100_20150317_005828	Check the Epitaph
-QUEST_LV_0100_20150317_005829	There are epitaphs about the mausoleum in Shirnas Plateau. Check the epitaph of Shirnas which is one of it. 
+QUEST_LV_0100_20150317_005829	There is an epitaph about the mausoleum in Shirnas Highland. Check the epitaph found there. 
```

Comment:
mausoleum -> Royal Mausoleum
epitaph -> Maybe an idea to change this to "inscription"? They're synonymous, but I think "inscription" is a more common and known term to users?